tfhe/src/shortint/ciphertext/mod.rs

```diff
@@ -85,6 +85,49 @@ pub struct Ciphertext {
     pub pbs_order: PBSOrder,
 }
 
+// #[cfg(feature = "safe_serialization")]
+impl crate::safe_serialization::ParameterSetConformant for Ciphertext {
+    type ParameterSet = super::PBSParameters;
+
+    fn conformant(&self, param: &Self::ParameterSet) -> bool {
+        let (expected_dim, message_modulus, carry_modulus, ciphertext_modulus);
+
+        match param {
+            super::PBSParameters::PBS(param) => {
+                expected_dim = match self.pbs_order {
+                    PBSOrder::KeyswitchBootstrap => {
+                        param.glwe_dimension.0 * param.polynomial_size.0
```

Comment:
LweDimenstion(param.glwe_dimension.0 * param.polynomial_size.0).to_lwe_size(), similar with the other pieces of code